CONTRIBUTING.md

```diff
@@ -4,17 +4,17 @@ Building Cabal for hacking
 --------------------------
 
 The current recommended way of developing Cabal is to use the
-`v2-build` feature which [shipped in cabal-install-1.24](http://blog.ezyang.com/2016/05/announcing-cabal-new-build-nix-style-local-builds/).
+`build` feature which [shipped in cabal-install-1.24](http://blog.ezyang.com/2016/05/announcing-cabal-new-build-nix-style-local-builds/).
```

Comment:
I think it can be cut down further: no need to mention a) that `build` is a feature (sounds weird: a build tool like cabal is supposed to have such "feature"!), b) when it was released, c) link to the blog post.